Formula/judy.rb

```diff
@@ -0,0 +1,56 @@
+class Judy < Formula
+  desc "State-of-the-art C library that implements a sparse dynamic array"
+  homepage "https://judy.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/judy/judy/Judy-1.0.5/Judy-1.0.5.tar.gz"
+  sha256 "d2704089f85fdb6f2cd7e77be21170ced4b4375c03ef1ad4cf1075bd414a63eb"
+  license "LGPL-2.1-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
```

Comment:
What is `pkg-config` needed for?

Reply:
that is mostly copy paste. I can remove it.